exampleSite/config-prod.toml

```diff
@@ -1,5 +1,4 @@
 languageCode = "en-us"
-title = "Massively"
```

Comment:
I assume this is a breaking change?

All existing instances of the Massively theme will need to update their `config.toml` files?

Not a problem if so, just will need to update the version number.

Reply:
Well, don't know exactly what config-prod.toml in examplesSite is for, I assumed it was in order to generate the example hugo site to showcase the theme in hugo theme's page.

In multilingual mode you can override params by language, so you can have for example a title for English and a title for Spanish, if no param is found in the current language being generated it defaults to the global params.

Perhaps I should leave title as is and only override it for Spanish just to make it clear to people using the theme that you can override it.

Reply:
@curttimson I reverted the change in config file regarding title, now it has a default value for English and is overriden in Spanish just to demonstrate how can be acheived.